<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrontPanel</class>
 <widget class="QMainWindow" name="FrontPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FrontPanel</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="pressureBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>551</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;MS Sans Serif&quot;;
</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="ventButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>140</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Sans Serif&quot;;</string>
     </property>
     <property name="text">
      <string>OPEN VENT</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="measurePressureDouble">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>230</y>
       <width>129</width>
       <height>22</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Sans Serif&quot;;</string>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-0.100000000000000</double>
     </property>
     <property name="maximum">
      <double>0.400000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.005000000000000</double>
     </property>
     <property name="value">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QPushButton" name="zeroPressure">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>70</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Sans Serif&quot;;</string>
     </property>
     <property name="text">
      <string>ZERO</string>
     </property>
    </widget>
    <widget class="QLabel" name="actualPressure">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>90</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Sans Serif&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="valveButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>210</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Sans Serif&quot;;
</string>
     </property>
     <property name="text">
      <string>OPEN VALVE</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="balanceButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Sans Serif&quot;;</string>
     </property>
     <property name="text">
      <string>2. Go to pressure</string>
     </property>
    </widget>
    <widget class="QLabel" name="PressureLabel">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>70</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Sans Serif&quot;;</string>
     </property>
     <property name="text">
      <string>Actual Pressure:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="initializeButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Sans Serif&quot;;</string>
     </property>
     <property name="text">
      <string>1. Initialize</string>
     </property>
    </widget>
    <widget class="QLabel" name="MeasurementPressureLabel">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>210</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Sans Serif&quot;;</string>
     </property>
     <property name="text">
      <string>Measure  Pressure (psi):</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>441</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1000</number>
     </property>
     <property name="maximum">
      <number>2000</number>
     </property>
     <property name="value">
      <number>1700</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="balancePressureDouble">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>160</y>
       <width>129</width>
       <height>22</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Sans Serif&quot;;
</string>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-0.100000000000000</double>
     </property>
     <property name="maximum">
      <double>0.400000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.005000000000000</double>
     </property>
     <property name="value">
      <double>-0.035000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="DesiredPressureLabel">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>140</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Sans Serif&quot;;</string>
     </property>
     <property name="text">
      <string>Desired  Pressure (psi):</string>
     </property>
    </widget>
    <widget class="QLabel" name="ActuatorPositionHeading">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>282</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Sans Serif&quot;;</string>
     </property>
     <property name="text">
      <string>Set Actuator Position</string>
     </property>
    </widget>
    <widget class="QPushButton" name="measureButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Sans Serif&quot;;</string>
     </property>
     <property name="text">
      <string>3. Measure</string>
     </property>
    </widget>
    <widget class="QPushButton" name="outIncrement">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>280</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>OUT</string>
     </property>
    </widget>
    <widget class="QPushButton" name="inIncrement">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>310</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>IN</string>
     </property>
    </widget>
    <widget class="QLabel" name="SystemStatusTextEdit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>381</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;MS Sans Serif&quot;;
color: rgb(255, 0, 0)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="SystemStatusLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;MS Sans Serif&quot;;</string>
     </property>
     <property name="text">
      <string>System Status:</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>490</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;MS Sans Serif&quot;;
color: red;</string>
    </property>
    <property name="text">
     <string>STOP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>490</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;MS Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>START </string>
    </property>
   </widget>
   <widget class="QLabel" name="FileToSaveHeading">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>Filename to save:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fileNameButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>430</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fileNameBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>429</y>
      <width>491</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>C:/Users/Admin/Desktop/Data/test</string>
    </property>
   </widget>
   <widget class="QLabel" name="comPortHeading">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>480</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>Connection Port:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="arduinoConnectBtn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>490</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>CONNECT</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comPortSelect">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>510</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>643</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
